src/loaders/jobs/taskfarmer/task_generator.py

```diff
@@ -83,6 +87,20 @@ def _pull_image(image_str, job_dir):
                              f"Standard error: {std_err.read()}")
 
 
+def _write_to_file(file_path, content, overwrite=True):
```

Comment:
Looks like none of the calls specified overwrite=False, so I assume that arg can be removed

Reply:
👍

Reply:
:+1: